params/params_testmixnet.go

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2017-2018 The qitmeer developers
+// Copyright (c) 2014-2016 The btcsuite developers
+// Copyright (c) 2015-2017 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package params
+
+import (
+    "github.com/Qitmeer/qitmeer/common"
+    "github.com/Qitmeer/qitmeer/core/protocol"
+    "github.com/Qitmeer/qitmeer/core/types/pow"
+    "math/big"
+    "time"
+)
+
+// testPowNetPowLimit is the highest proof of work value a block can
+// have for the test network. It is the value 2^232 - 1.
+var	testPowNetPowLimit = new(big.Int).Sub(new(big.Int).Lsh(common.Big1, 232), common.Big1)
+
+// testPowNetParams defines the network parameters for the test network.
+var TestPowNetParams = Params{
+    Name:        "testmixnet",
+    Net:         protocol.MixTestNet,
+    DefaultPort: "18132",
+    DNSSeeds: []DNSSeed{
+        {"testPowNet-seed.hlcwallet.info", true},
+        {"testPowNet-seed.qitmeer.xyz", true},
+        {"testPowNet-seed.qitmeer.top", true},
+    },
+
+    // Chain parameters
+    GenesisBlock:             &testPowNetGenesisBlock,
+    GenesisHash:              &testPowNetGenesisHash,
+    ReduceMinDifficulty:      false,
+    MinDiffReductionTime:     0, // Does not apply since ReduceMinDifficulty false
+    GenerateSupported:        true,
+    PowConfig :&pow.PowConfig{
+        Blake2bdPowLimit:                 testPowNetPowLimit,
+        Blake2bdPowLimitBits:             0x1e00ffff,
+        Blake2bDPercent:          34,
+        CuckarooPercent:          33,
+        CuckatooPercent:          33,
+        CuckarooDiffScale:            1856,
+        CuckatooDiffScale:            1856,
+        CuckarooMinDifficulty:     1000,
+        CuckatooMinDifficulty:     1000,
+    },
+
+    WorkDiffAlpha:            1,
+    WorkDiffWindowSize:       60,
+    WorkDiffWindows:          20,
+    MaximumBlockSizes:        []int{1310720},
+    MaxTxSize:                1000000,
+    TargetTimePerBlock:       time.Minute * 1,
+    TargetTimespan:           time.Minute * 1 * 1, // TimePerBlock * WindowSize
+    RetargetAdjustmentFactor: 3,
+
+    // Subsidy parameters.
+    BaseSubsidy:              2500000000, // 25 Coin
+    MulSubsidy:               100,
+    DivSubsidy:               101,
+    SubsidyReductionInterval: 2048,
+    WorkRewardProportion:     9,
+    StakeRewardProportion:    0,
+    BlockTaxProportion:       1,
+
+    // Checkpoints ordered from oldest to newest.
+    Checkpoints: []Checkpoint{
+    },
+
+    // Consensus rule change deployments.
+    //
+    Deployments: map[uint32][]ConsensusDeployment{
+    },
+
+    // Address encoding magics
+    NetworkAddressPrefix: "T",
```

Comment:
I suggest eliminating the additional `testmixnet` param. If you really want this one, Please provide all necessary configurations included all address prefix etc. such as 
```golang
NetworkAddressPrefix: "X",
```

Reply:
add new network prefix